### PR TITLE
Fix Discovery Linters

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This PR fixes some checks for the new listing rules

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: pkg/consts/crdClient.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package consts
+
+const Apis = "/apis"
```
Comment (giorio94) [inline pkg/consts/crdClient.go:3]:
```suggestion
const ApisPath = "/apis"
```

File: internal/discovery/foreign-cluster-operator/auth.go [hunk 2]
```diff
@@ -11,19 +11,19 @@ import (
...
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	client_scheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog"
-
-	discoveryv1alpha1 "github.com/liqotech/liqo/apis/discovery/v1alpha1"
-	"github.com/liqotech/liqo/pkg/auth"
-	"github.com/liqotech/liqo/pkg/crdClient"
-	"github.com/liqotech/liqo/pkg/discovery"
-	"github.com/liqotech/liqo/pkg/kubeconfig"
```
Comment (giorio94) [inline internal/discovery/foreign-cluster-operator/auth.go:26]: This should be reverted

File: .golangci.yml [hunk 3]
```diff
@@ -107,3 +108,6 @@ issues:
     - linters:
         - govet
       text: 'declaration of "(err|ctx)" shadows declaration at'
+    - linters:
+        - gosec
+      text: TLS InsecureSkipVerify set true.
```
Comment (giorio94) [inline .golangci.yml:113]: It would be better if there were a comment explaining why this is needed

File: pkg/discovery/const.go [hunk 4]
```diff
@@ -1,44 +1,67 @@
...
 const (
+	// TenantControlNamespaceLabel used to mark the tenant control namespaces
 	TenantControlNamespaceLabel = "discovery.liqo.io/tenant-control-namespace"
-	ClusterRoleLabel            = "discovery.liqo.io/cluster-role"
+	// ClusterRoleLabel used all the Liqo cluster roles
+	ClusterRoleLabel = "discovery.liqo.io/cluster-role"
 
-	ClusterIdLabel      = "discovery.liqo.io/cluster-id"
-	AuthTokenLabel      = "discovery.liqo.io/auth-token"
+	// ClusterIDLabel
```
Comment (giorio94) [inline pkg/discovery/const.go:9]: The comment is empty

**Pull request merged**